py-polars/polars/internals/lazy_frame.py

```diff
@@ -1144,10 +1125,7 @@ def head(self, n: int = 5) -> "LazyFrame":
         --------
 
         >>> df = pl.DataFrame(
-        ...     {
```

Comment:
Also when creating a `DataFrame`, I think the multiline version is more readable. You clearly see what the columns are.

Reply:
Yes, that is why I forced them multiline. Else the user already has to print the dataframe to see the columns.